app/controllers/api/v1/code_schools_controller.rb

```diff
@@ -2,7 +2,8 @@ module Api
   module V1
     class CodeSchoolsController < ApplicationController
       def index
-        render json: CodeSchools.all
+        @schools = CodeSchool.all
```

Comment:
Since we are not passing this instance variable to a Rails view, you should be able to truncate these two lines into:

```ruby
render json: CodeSchool.all
```